charts/pega/README.md

```diff
@@ -658,6 +658,21 @@ If you are planning to use Cassandra (usually as a part of Pega Customer Decisio
 
 To use an existing Cassandra deployment, set `cassandra.enabled` to `false` and configure the `dds` section to reference your deployment.
 
+The following parameters are available for configuration of the connection to the external Cassandra cluster.
```

Comment:
Using the following available parameters to configure the connection to your external Cassandra cluster.

Reply:
I will make this change